spec/paperclip/io_adapters/registry_spec.rb

```diff
@@ -4,7 +4,7 @@
   context "for" do
     before do
       class AdapterTest
-        def initialize(target); end
+        def initialize(target, _ = {}); end
```

Comment:
Unused method argument - `target`. If it's necessary, use `_` or `_target` as an argument name to indicate that it won't be used. You can also write as `initialize(*)` if you want the method to accept any arguments but don't care about them.